test/scripts/e2e-kind.sh

```diff
@@ -142,4 +142,5 @@ ginkgo --nodes=${NUM_NODES} \
 	--provider=local \
 	--dump-logs-on-failure=false \
 	--report-dir=${E2E_REPORT_DIR}	\
-	--disable-log-dump=true
+	--disable-log-dump=true \
+	--num-nodes=${NUM_NODES}
```

Comment:
hmm, be careful, there are 2 meanings for nodes here, one are the ginkgo nodes , that are the ones that ginkgo uses for parallelize the execution , check line 135 `--nodes=${NUM_NODES}`

Let´s name this NUM_WORKER_NODES or something similar

In parallel, that logic of the e2e framework has to be improved ...

Reply:
ACK, will use a separate environment variable for `--num-nodes`